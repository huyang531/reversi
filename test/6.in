fe
df
cd
ge
ff
fd
fc
dc
ce
ec
cc
fb
fa
bd
ad
ac
ab
gd
bc
gf
ae
fg
hd
hc
hb
eb
gc
db
da
be
ea
ca
ba
cb
ef
eg
af
cf
bf
gb
ga
he
hf
hg
gg
eh
dh
ha
fh
hh
gh
ch
dg
cg
bh
ag
ah
bb
aa
bg
