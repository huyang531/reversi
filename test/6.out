  abcdefgh
a ........
b ........
c ........
d ...WB...
e ...BW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ........
d ...WB...
e ...BB...
f ....B...
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ........
d ...WWW..
e ...BB...
f ....B...
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ...B....
d ...BWW..
e ...BB...
f ....B...
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ...B....
d ...BWW..
e ...BW...
f ....W...
g ....W...
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ...B....
d ...BWW..
e ...BB...
f ....WB..
g ....W...
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ...B....
d ...BWW..
e ...BW...
f ...WWB..
g ....W...
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ...B....
d ...BWW..
e ...BW...
f ..BBBB..
g ....W...
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ...B....
d ..WWWW..
e ...BW...
f ..BBBB..
g ....W...
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ...BB...
d ..WWBW..
e ...BB...
f ..BBBB..
g ....W...
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ...BB...
d ..WWBW..
e ..WBB...
f ..BWBB..
g ....W...
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..BBB...
d ..BBBW..
e ..BBB...
f ..BWBB..
g ....W...
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..BBB...
d ..BBBW..
e ..BBB...
f .WWWBB..
g ....W...
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..BBB...
d ..BBBW..
e ..BBB...
f BBBBBB..
g ....W...
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ...W....
c ..BBW...
d ..BBBW..
e ..BBB...
f BBBBBB..
g ....W...
h ........
Enter move for colour B (RowCol):   abcdefgh
a ...B....
b ...B....
c ..BBW...
d ..BBBW..
e ..BBB...
f BBBBBB..
g ....W...
h ........
Enter move for colour W (RowCol):   abcdefgh
a ..WB....
b ...W....
c ..BBW...
d ..BBBW..
e ..BBB...
f BBBBBB..
g ....W...
h ........
Enter move for colour B (RowCol):   abcdefgh
a .BBB....
b ...W....
c ..BBW...
d ..BBBW..
e ..BBB...
f BBBBBB..
g ....W...
h ........
Enter move for colour W (RowCol):   abcdefgh
a .BBB....
b ...W....
c ..BWW...
d ..BWBW..
e ..BWB...
f BBBWBB..
g ...WW...
h ........
Enter move for colour B (RowCol):   abcdefgh
a .BBB....
b ..BW....
c ..BBW...
d ..BWBW..
e ..BWB...
f BBBWBB..
g ...WW...
h ........
Enter move for colour W (RowCol):   abcdefgh
a .BBB....
b ..BW....
c ..BBW...
d ..BWBW..
e ..BWB...
f BBBWWB..
g ...WWW..
h ........
Enter move for colour B (RowCol):   abcdefgh
a .BBBB...
b ..BB....
c ..BBW...
d ..BWBW..
e ..BWB...
f BBBWWB..
g ...WWW..
h ........
Enter move for colour W (RowCol):   abcdefgh
a .BBBB...
b ..BB....
c ..BBW...
d ..BWBW..
e ..BWB...
f BBBWWWW.
g ...WWW..
h ........
Enter move for colour B (RowCol):   abcdefgh
a .BBBB...
b ..BB....
c ..BBW...
d ..BBBW..
e ..BBB...
f BBBBWWW.
g ...BWW..
h ...B....
Enter move for colour W (RowCol):   abcdefgh
a .BBBB...
b ..BB....
c ..BBW...
d ..BBBW..
e ..BBB...
f BBBBWWW.
g ...WWW..
h ..WB....
Enter move for colour B (RowCol):   abcdefgh
a .BBBB...
b ..BB....
c ..BBW...
d ..BBBW..
e ..BBB...
f BBBBWWW.
g ...WWW..
h .BBB....
Enter move for colour W (RowCol):   abcdefgh
a .BBBB...
b ..BB....
c ..BBW...
d ..BBBW..
e .WBBB...
f BBWBWWW.
g ...WWW..
h .BBB....
Enter move for colour B (RowCol):   abcdefgh
a .BBBB...
b ..BB....
c ..BBW...
d ..BBBW..
e .WBBB...
f BBBBWWW.
g ..BWWW..
h .BBB....
Enter move for colour W (RowCol):   abcdefgh
a .BBBB...
b ..BB....
c ..BBW...
d .WWWWW..
e .WWBB...
f BBBWWWW.
g ..BWWW..
h .BBB....
Enter move for colour B (RowCol):   abcdefgh
a .BBBB...
b ..BB....
c ..BBW...
d BWWWWW..
e .BWBB...
f BBBWWWW.
g ..BWWW..
h .BBB....
Enter move for colour W (RowCol):   abcdefgh
a .BBBB...
b ..BBW...
c ..BWW...
d BWWWWW..
e .BWBB...
f BBBWWWW.
g ..BWWW..
h .BBB....
Enter move for colour B (RowCol):   abcdefgh
a .BBBB...
b ..BBW...
c ..BWW...
d BBWWWW..
e BBWBB...
f BBBWWWW.
g ..BWWW..
h .BBB....
Enter move for colour W (RowCol):   abcdefgh
a .BBBB...
b ..BBW...
c W.BWW...
d BWWWWW..
e BBWBB...
f BBBWWWW.
g ..BWWW..
h .BBB....
Enter move for colour B (RowCol):   abcdefgh
a .BBBB...
b B.BBW...
c B.BWW...
d BWWWWW..
e BBWBB...
f BBBWWWW.
g ..BWWW..
h .BBB....
Enter move for colour W (RowCol):   abcdefgh
a .BBBB...
b B.BBW...
c BWWWW...
d BWWWWW..
e BBWBB...
f BBBWWWW.
g ..BWWW..
h .BBB....
Enter move for colour B (RowCol):   abcdefgh
a .BBBB...
b B.BBW...
c BWWBW...
d BWWWBW..
e BBWBBB..
f BBBWBWW.
g ..BBWW..
h .BBB....
Enter move for colour W (RowCol):   abcdefgh
a .BBBB...
b B.BBW...
c BWWBW...
d BWWWBW..
e BBWWWWW.
f BBBWBWW.
g ..BBWW..
h .BBB....
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b B.BBB...
c BWWBW...
d BWWWBW..
e BBWWWWW.
f BBBWBWW.
g ..BBWW..
h .BBB....
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b B.BBB...
c BWWBWW..
d BWWWWW..
e BBWWWWW.
f BBBWBWW.
g ..BBWW..
h .BBB....
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW..
e BBBWWWW.
f BBBWBWW.
g ..BBWW..
h .BBB....
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW..
e BWBWWWW.
f BWWWBWW.
g .WWWWW..
h .BBB....
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW..
e BWBWWWW.
f BBWWBWW.
g BWWWWW..
h .BBB....
W player has no valid move.
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW..
e BBBWWWW.
f BBBWBWW.
g BWWBBW..
h .BBBB...
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW..
e BBBWWWW.
f BBBWBWW.
g BWWBWW..
h .BBBBW..
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW..
e BBBWWWW.
f BBBWBWW.
g BWWBWB..
h .BBBBBB.
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW..
e BBBWWWW.
f BBBWBWW.
g BWWBWWW.
h .BBBBBB.
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW..
e BBBBBBBB
f BBBWBWB.
g BWWBWBW.
h .BBBBBB.
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW.W
e BBBBBBWB
f BBBWBWB.
g BWWBWBW.
h .BBBBBB.
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW.W
e BBBBBBWB
f BBBWBWB.
g BBWBWBW.
h BBBBBBB.
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW.W
e BBBBBBWW
f BBBWBWWW
g BBWBWBW.
h BBBBBBB.
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW.W
e BBBBBBWW
f BBBWBBWW
g BBWBWBB.
h BBBBBBBB
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW..
d BWWBWW.W
e BBBBBBWW
f BBBWBBWW
g BBWBWWWW
h BBBBBBBB
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBW.B
d BWWBWW.B
e BBBBBBWB
f BBBWBBWB
g BBWBWWWB
h BBBBBBBB
W player has no valid move.
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWBBB.B
d BWWBBBBB
e BBBBBBBB
f BBBWBBBB
g BBWBWWBB
h BBBBBBBB
Enter move for colour W (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB..
c BWWWWWWB
d BWWBBWBB
e BBBBWBBB
f BBBWBBBB
g BBWBWWBB
h BBBBBBBB
Enter move for colour B (RowCol):   abcdefgh
a .BBBBB..
b B.BBBB.B
c BWWWWWBB
d BWWBBBBB
e BBBBBBBB
f BBBBBBBB
g BBBBWWBB
h BBBBBBBB
Enter move for colour W (RowCol):   abcdefgh
a .BBBBBW.
b B.BBBW.B
c BWWWWWBB
d BWWBBBBB
e BBBBBBBB
f BBBBBBBB
g BBBBWWBB
h BBBBBBBB
Enter move for colour B (RowCol):   abcdefgh
a .BBBBBBB
b B.BBBW.B
c BWWWWWBB
d BWWBBBBB
e BBBBBBBB
f BBBBBBBB
g BBBBWWBB
h BBBBBBBB
Enter move for colour W (RowCol):   abcdefgh
a .BBBBBBB
b BWWWWW.B
c BWWWWWBB
d BWWBBBBB
e BBBBBBBB
f BBBBBBBB
g BBBBWWBB
h BBBBBBBB
Enter move for colour B (RowCol):   abcdefgh
a BBBBBBBB
b BBWWWW.B
c BWBWWWBB
d BWWBBBBB
e BBBBBBBB
f BBBBBBBB
g BBBBWWBB
h BBBBBBBB
W player has no valid move.
Enter move for colour B (RowCol):   abcdefgh
a BBBBBBBB
b BBBBBBBB
c BWBWWBBB
d BWWBBBBB
e BBBBBBBB
f BBBBBBBB
g BBBBWWBB
h BBBBBBBB
W player has no valid move.
B player has no valid move.
Black wins by 50 points!
